tests/parallel/direct-mesh-access/AccessReceivedMeshAndMapping.cpp

```diff
@@ -9,6 +9,12 @@
 // by another participant (see above). In addition to the direct mesh access
 // and data writing in one direction, an additional mapping (NN) is defined
 // in the other direction.
+// We also test that getMeshVertexSize and getMeshVertexIDsAndCoordinates
+// correctly filter out one vertex (-0.5, -0.5) which is not in the access
+// region, but is used for mapping.
+// Since the user has no opportunity to get the "filtered" vertex (unless
+// the bounding box is enlarged), the read data value on the SolverTwo
+// participant is zero ( last entry in l115 {15, 16, 0}).
```

Comment:
FYI @uekerman it might happen that vertices are now essentially not visible to the users or filtered out, but I guess that's a logical consequence of how we do now things.

Reply:
The user gets a warning if vertices are filtered out completely during the repartitioning. That's here, however, not the case. We also can't give a warning, as anyway vertices are outside the box due to the halo-region required for the just-in-time mapping.

Reply:
OK, then we did what we could.